torch/_inductor/graph.py

```diff
@@ -1532,7 +1532,8 @@ def codegen_with_cpp_wrapper(self):
         if "cuda" in self.device_types:
             # first pass
             self.cpp_wrapper = False
-            compiled = self.compile_to_module().call
+            with config.patch({"triton.store_cubin": True}):
+                compiled = self.compile_to_module().call
```

Comment:
Being curious, why do we need this very special handling here? Can we add some comments in the code? Thanks.

Reply:
It's because of the difference between AOT and JIT. In AOT, we set store_cubin before compiling forward, but in JIT, we lost that when compiling backward.

I will add comments, maybe also consolidating the logic to here.